### DISCUSSION TITLE
Time series classification

### DISCUSSION THREAD

**Discussion by Glaswartron** [autogluon/autogluon#3044] [Q&A] [answered]
Hi everyone,

is it possible to use AutoGluon for time series classification?
My dataset consists of time series with a categorical label for each series and I want to
predict the label based on the entire time series. 
The tutorials and documentation I saw only cover forecasting, but I was wondering 
whether classifying / categorizing time series is also possible or maybe planned for future versions?

Thanks in advance.

**Comment by shchur** [comment, marked as answer]
You can convert each time series into a fixed-dimensional feature vector using libraries like [tsfresh](https://tsfresh.readthedocs.io/en/latest/index.html) or [catch22](https://github.com/DynamicsAndNeuralSystems/catch22), and then use the `autogluon.tabular.TabularPredictor` with `problem_type="classification"` to classify time series.
